templates/components/templates/page-layout--three-column/page-layout--three-column.twig

```diff
@@ -0,0 +1,14 @@
+<div class="page-layout page-layout--three-column">
+  <div class="column column--head">
+    {{ headContent }}
```

Comment:
Twig variable naming conventions use "_" separators and not camel case.

Reply:
Thanks for the heads_up!  Fixed.